src/components/Header/Header.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  HeaderWrapper,
+  LogoContainer,
+  LinksContainer,
+  Link,
+} from './Header.style';
+
+function Header({ logo, links }) {
+  return (
+    <HeaderWrapper>
+      <LogoContainer>
+        <Link href="/">
+          <img src={logo} alt="Logo" />
+        </Link>
+      </LogoContainer>
+      <LinksContainer>
+        {links.map((item) => (
+          <Link key={`${item.title}-${Math.random() * 999}`} href={item.link}>{item.title}</Link>
```

Comment:
As a key either title or link should be enough. 

Fyi: Using a random key as you did here can break the app if the list items you render change their order. The key should be constant per item